redstone/client.py

```diff
@@ -605,6 +605,31 @@ def rotate(self, key_id, payload=None):
 
         return self._action(key_id, "rotate", data)
 
+    def get_registrations(self, key_id: str = None, crn: str = None):
+        """
+        Retrieve a list of registrations
+
+        If `key_id` is None (the default) all registrations for the instance are
+        returned, otherwise only the registrations associated with a specified root
+        key are returned.
```

Comment:
What is the CRN parameter used for then?

Reply:
There is API Docs link at the class level.

    """
    API Docs: https://cloud.ibm.com/apidocs/key-protect
    """

Reply:
Add API doc links at function level as well